### DISCUSSION TITLE
What I need know if I want open and keep a site in the hyper web?

### DISCUSSION THREAD

**Discussion by marcussacana** [beakerbrowser/beaker#1854] [Q&A]
I'm very new to the beaker and I still need to learn how the things works for the creators.
And my doubts are basically:

Q: ~I can update my site after published?~
A: After research looks yes, I can create a fork, modify then give a merge... not sure if have a better method by the way.

Q: How can I ensure the owership of my site in cases like when we need to format the computer?
N: Like backup a key or anything like that...

Q: Is possible create a 404 page?

Q: Any other detail that I don't know and I must keep attention?

**Comment by alizehctl** [comment]
> Q: How can I ensure the owership of my site in cases like when we need to format the computer?
> N: Like backup a key or anything like that...

When I had to switch laptops, I backed up my `~/.hyperdrive` folder and the `/Library/Application Support/Beaker Browser` folder. By copying these folders back to their locations on your new OS, you should be able to retain your Beaker profile. On Linux, these folders live at `~/.hyperdrive` and `~/.config/Beaker Browser`, respectively.

**Reply by marcussacana** [reply]
Thanks!

Other Question: If for some reason I create the site and backup this 2 folders, then after some changes, the old backup will be able to keep my owership? or will fail due be outdated?

**Reply by alizehctl** [reply]
You'll still have your profile, but you'll lose the changes you made since the old backup. (not sure though, perhaps @pfrazee can confirm)

**Reply by pfrazee** [reply]
@aadilayub is right about what you can do, but right now the protocol isn't smart enough to deal with 2 computers publishing to the same hyperdrive, so you gotta be careful to **move** the data and not just **copy** it. If you publish from 2 computers, you might corrupt your drives. (This is something that's being worked on actively by the protocol team now.)

**Comment by pfrazee** [comment]
Q: Is possible create a 404 page?
A: Check out https://docs.beakerbrowser.com/developers/frontends-.ui-folder
